### ISSUE TITLE
Add definition list to search result page component

### ISSUE THREAD

**Issue by nikdragovic** [emory-libraries/dlp-lux#655]
The second-row element is in app/views/catalog/_index_header.html.erb

## Acceptance Criteria
- [x] Wrap list currently in document-heading-second-row in a definition list (dl)element so that screen readers can properly announce them.
- [x] If this is modifying default Blacklight behavior, report it back to the community.

**Comment by nikdragovic** [comment]
@bwatson78 This appears to be what is flagged, from what I can tell. I'm not sure how this would be defined if so; it's just empty space.
![Screen Shot 2020-07-22 at 10.40.44 AM.png](https://images.zenhubusercontent.com/5ba135cf945aef49576d50f2/1c88920d-8c91-4bf1-9a65-10c7cba133e1)

**Comment by bwatson78** [comment]
@nikdragovic and @eporter23: This isn't blacklight standard behavior, rather our custom iterations.

**Comment by bwatson78** [comment]
PR made: https://github.com/emory-libraries/dlp-lux/pull/663

![Screen Shot 2020-07-23 at 8.08.22 AM.png](https://images.zenhubusercontent.com/5e6f6c00c3241244877b4aea/d5bf9e87-4bed-454f-bc3a-343bf2a0d202)
